modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js

```diff
@@ -237,19 +242,23 @@ export default class NestedTabs extends Base {
 	}
 
 	getHeadingEvents() {
+		const slider = this.elements.$headingContainer[ 0 ];
+
 		return {
-			mousedown: this.changeScrollStatus.bind( this ),
-			mouseup: this.changeScrollStatus.bind( this ),
-			mouseleave: this.changeScrollStatus.bind( this ),
-			mousemove: this.setHorizontalTabTitleScrollValues.bind( this ),
+			mousedown: changeScrollStatus.bind( this, slider ),
+			mouseup: changeScrollStatus.bind( this, slider ),
+			mouseleave: changeScrollStatus.bind( this, slider ),
+			mousemove: setHorizontalTitleScrollValues.bind( this, slider, this.getHorizontalScrollSetting() ),
 		};
 	}
 
 	bindEvents() {
 		this.elements.$tabTitles.on( this.getTabEvents() );
 		this.elements.$headingContainer.on( this.getHeadingEvents() );
 
-		this.resizeListenerNestedTabs = this.setHorizontalScrollAlignment.bind( this );
+		const slider = this.elements.$headingContainer[ 0 ];
+
+		this.resizeListenerNestedTabs = setHorizontalScrollAlignment.bind( this, slider, this.getTabsDirection(), '--n-tabs-heading-justify-content', this.getHorizontalScrollSetting() );
```

Comment:
This looks a bit overcomplicated, but I am not sure if there is another solution.